### PR TITLE
Update Bower manifest

### PR DESCRIPTION
**Pull request by simison**
- Remove deprecated "version": http://bower.io/docs/creating-packages/#version
- Add "license"
- Add "homepage"
- Use fuzzier version requirement for angular-messages (~)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -12,7 +13,7 @@
     "angular-ui-utils": "bower",
     "angular-ui-router": "~0.2",
     "angular-file-upload": "1.1.5",
-    "angular-messages": "1.3.17",
+    "angular-messages": "~1.3",
```
Comment (codydaig) [inline bower.json]: Can we leave the .17 so we know what version it was brought in and tested with? That's helpful when a new version is released and it contains some random bug and we need to revert.

**Comment by codydaig** [comment]
@simison Left a line comment. Otherwise looks good.

File: bower.json (hunk 1 above)
Comment (simison) [inline bower.json]: I don't mind, tho this goes quite much into bikeshedding category anyway. :-)

Angular-messages is a core module, so it should be pretty solid and reliable for minor 1.3.x updates, just like the rest of the angular- packages listed there. Just sayin'.

Comment (simison) [inline bower.json]: Added.

Comment (codydaig) [inline bower.json]: :+1: Squash and I'll give my approval. Thanks! :)

**Comment by simison** [comment]
@codydaig Squashed.

**Comment by codydaig** [comment]
LGTM!

@simison Thanks! :-D

**Comment by mleanos** [comment]
LGTM

**Comment by lirantal** [comment]
LGTM

**Pull request merged**